apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -79,6 +79,16 @@ type StorageSizes struct {
 	PerWorkspace *resource.Quantity `json:"perWorkspace,omitempty"`
 }
 
+//TODO: Should this be here or in another file?
+// Allows specifying an external DevWorkspace-Operator configuration
+// which will merged with the workspace's DevWorkspace-Operator configuration.
+type ExternalConfig struct {
+	// The metadata.name of the external DevWorkspace-Operator configuration.
+	Name string `json:"name,omitempty"`
+	// The metadata.namespace of the external DevWorkspace-Operator configuration.
+	Namespace string `json:"namespace,omitempty"`
+}
+
```

Comment:
This could just be unmarshalled to a `types.NamespacedName` I believe, removing the need for a specific struct (haven't tested, though).

If it is needed after all, this struct should be defined elsewhere (perhaps in the constants/annotations package?)

Reply:
`types.NamespacedName` seems like it'd work perfectly :) I'll give that a try